src/krux/pages/pub_key_view.py

```diff
@@ -36,17 +36,26 @@
 
 
 class PubkeyView(Page):
-    """UI to show and export extended public key"""
+    """
+    UI to show and export extended public key
+
+    :param ctx: :class:`krux.pages.Page`
+    """
 
     def __init__(self, ctx):
         super().__init__(ctx, None)
         self.ctx = ctx
         self.utils = Utils(self.ctx)
 
+    # TODO: Is it worth to define the functions inside as protected members of PubkeyView?
+    # (to aim a better readability)
```

Comment:
I think this is a discussion that may happen on a discussion or an issue, not on the code

Reply:
I put this because pylint warns

Reply:
I called `protected` in the definition used by `pylint`, those object methods that starts `_`. I was following the pylint standards.